services/dashboard/views/dashboard.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+'use strict';
+
+angular
+  .module('dashboard', [
+    'ngRoute',
+    'ng-breadcrumbs',
+    'smart-table',
+    'filter.readable',
+    'dashboard.restapi',
+    'dashboard.streamingservice',
+    'dashboard.cluster',
+    'dashboard.apps',
+    'dashboard.apps.appmaster'
+  ])
+
+  .config(['$routeProvider', function ($routeProvider) {
+    $routeProvider
+      // TODO: we need a real landing page
+      .when('/', {redirectTo: '/cluster', label: 'Home'})
+      .otherwise({redirectTo: '/'});
+  }])
+
+  .constant('conf', {
+    debug: false,
+    restapiAutoRefreshInterval: 2000,
+    restapiRoot: '',
+    webSocketUri: 'ws://' + document.location.hostname + ':8091/',
```

Comment:
Add TODO here for REST service call that returns this information. Something like
//TODO Fetch WS URI's. See #546